content/docs/state-and-lifecycle.md

```diff
@@ -407,41 +408,41 @@ Then you can update them independently with separate `setState()` calls:
   }
 ```
 
-The merging is shallow, so `this.setState({comments})` leaves `this.state.posts` intact, but completely replaces `this.state.comments`.
+Sammanslagningen är grund, så `this.setState({comments})` lämnar `this.state.posts` intakt, men ersätter helt` this.state.comments`.
 
-## The Data Flows Down {#the-data-flows-down}
+## Data flödar nedåt {#the-data-flows-down}
 
-Neither parent nor child components can know if a certain component is stateful or stateless, and they shouldn't care whether it is defined as a function or a class.
+Varken föräldrar eller barnkomponenter kan veta om en viss komponent har state eller inte, och de borde inte bry sig om den definieras som en funktion eller en klass.
 
-This is why state is often called local or encapsulated. It is not accessible to any component other than the one that owns and sets it.
+Det är därför staten ofta kallas lokal eller inkapslad. Den är inte tillgänglig för någon annan komponent än den som äger och ställer in den.
```

Comment:
`Det är därför staten ofta kallas lokal eller inkapslad. Den är inte tillgänglig för någon annan komponent än den som äger och ställer in den.`

->

`Det är därför state ofta kallas lokalt eller inkapslat. Det är inte tillgängligt för någon annan komponent än den som äger och ställer in det.`